source/guide/3.2/plugin-hostpool.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: bt_wiki
+title: Host Pool Plugin
+category: Plugins
+publish: true
+abstract: Cloudify Host Pool plugin description and configuration
+pageord: 600
+---
+{%summary%}
+{%endsummary%}
+
+
+# Description
+
+The Host Pool plugin is an interface to [Host Pool service](https://github.com/cloudify-cosmo/cloudify-host-pool-service).
+
+Host Pool Service allows to use an existing environment infrastructure for deploying services and applications.
+
+
+# Plugin Requirements
+
+* Python Versions:
+    * 2.7.x
+
+
+# Types
+
+## cloudify.hostpool.nodes.Host
+
+**Derived From:** [cloudify.nodes.Compute](reference-types.html)
+
+**Mapped Operations:**
+
+  * `cloudify.interfaces.lifecycle.start` acquires the host from the Host Pool service and sets runtime properties.
+  * `cloudify.interfaces.lifecycle.delete` releases the host from the Host Pool service.
+
+Runtime properties set in `cloudify.interfaces.lifecycle.start` operation are provided as inputs to the following agent worker installer operations:
```

Comment:
same here, you can avoid duplication by just specifying operation inputs description once and say its the same for all operations. here you should describe the username, password etc...